hawkular-alerts-rest-tests/src/test/groovy/org/hawkular/alerts/rest/ActionsITest.groovy

```diff
@@ -333,9 +333,7 @@ class ActionsITest extends AbstractITestBase {
         assertEquals(1, resp.data.size())
 
         // ENABLE Trigger
-        testTrigger.setEnabled(true);
-
-        resp = client.put(path: "triggers/test-email-threshold", body: testTrigger)
+        resp = client.put(path: "triggers/enabled", query:[triggerIds:"test-email-threshold",enabled:true] )
```

Comment:
We should maintain tests with the old way to disable (using the whole trigger update) to not miss possible regressions.
I see there were several modifications in tests, I couldn't see if all of them are updated with this one.
Let's maintain explicit tests for non-groups and groups triggers.